File: chamomile.web/src/model/GeneralStatistics.ts
export default interface GeneralStatistics {
    minTs: string
    maxTs: string
    favCount: number
    upscaledCount: number
    downloadCount: number
    totalDownloads: number
    totalCount: number
    avgGenTime: number
    maxImageId: number
    countBySource: Record<string, number>
}

File: Chamomile.Common/GeneralStatistics.cs
namespace Chamomile.Common {
    public class GeneralStatistics {
        public DateTime? MinTs { get; set; }
        public DateTime? MaxTs { get; set; }
        public int FavCount { get; set; }
        public int UpscaledCount { get; set; }
        public int DownloadCount { get; set; }
        public int? TotalDownloads { get; set; }
        public int TotalCount { get; set; }
        public double AvgGenTime { get; set; }
        public double MaxImageId { get; set; }

        public Dictionary<string,int> CountBySource { get; set; }

    }
}
